packages/frontend/src/routes/constants.js

```diff
@@ -0,0 +1,7 @@
+const GAME_STAGES = Object.freeze({
+  SETUP: 'SETUP',
+  PLAYING: 'PLAYING',
+  GAME_OVER: 'GAME_OVER',
+});
+
+export { GAME_STAGES };
```

Comment:
I feel like we'll eventually have other constants that aren't routes. I think we should have a directory called `constants`, which would have an `index.js` file which would import `routes` from `constants/routes.js` and export it, so routes would be imported like `import { routes } from 'constants'`

Reply:
Totally makes sense. Made that change.